app/build.gradle.kts

```diff
@@ -39,15 +39,15 @@ dependencies {
     implementation(libs.androidx.core.ktx)
     implementation(libs.androidx.lifecycle.runtime.ktx)
     implementation(libs.androidx.activity.compose)
-    implementation(platform(libs.androidx.compose.bom))
+    implementation(platform(libs.chrisbanes.androidx.compose.bom))
     implementation(libs.androidx.compose.ui)
     implementation(libs.androidx.compose.ui.graphics)
     implementation(libs.androidx.compose.ui.tooling.preview)
     implementation(libs.androidx.compose.material3)
     testImplementation(libs.junit4)
     androidTestImplementation(libs.androidx.junit)
     androidTestImplementation(libs.androidx.espresso.core)
-    androidTestImplementation(platform(libs.androidx.compose.bom))
+    androidTestImplementation(platform(libs.chrisbanes.androidx.compose.bom))
```

Comment:
Curious, what's the difference?

Reply:
ah, yeah, so I had run into an issue when using the new type safe compose navigation library since it introduced some version conflicts and because both that and the text field state are not yet stable. So using the official Google BOM defaults everything to the latest stable versions, this BOM from Chris Banes uses the alpha versions instead. It's better explained here :simple_smile: https://stackoverflow.com/questions/78499314/on-android-new-version-of-navigation-compose-2-8-0-alpha08-and-up-not-compati